src/decaylanguage/utils/utilities.py

```diff
@@ -45,12 +45,13 @@ def split(x):  # type: ignore [no-untyped-def]
     return out
 
 
-def filter_lines(matcher, inp):  # type: ignore [no-untyped-def]
+def filter_lines(
+    matcher: Pattern[str], inp: List[str]
+) -> Tuple[List[Dict[str, Union[str, Any]]], List[str]]:
```

Comment:
Adding type hints here, since they were missing. If you want to see the type hints for the standard library, like `re`, those are at https://github.com/python/typeshed/tree/master/stdlib - checking that is very helpful. That's where I found `Pattern`, and that it was in `typing`.

Reply:
Thank you for the pointer!